src/styles/modules/nav.scss

```diff
@@ -0,0 +1,20 @@
+.nav {
+   display: flex;
+   justify-content: center; 
+   box-sizing: border-box;
+   background-color: #313030;
+}
+
+.nav-item { 
+   margin: 0 58px 0 58px;
```

Comment:
likely you don't need `margin` styles, when I hover over your `.nav-item` I see red background appears at top and bottom but not on sides, you have to use only `padding: 34px 58px` and thats all